mlflow/server/__init__.py

```diff
@@ -8,6 +8,7 @@
 from mlflow.server.handlers import get_artifact_handler, STATIC_PREFIX_ENV_VAR, _add_static_prefix
 from mlflow.utils.process import exec_cmd
 
+
```

Comment:
Problably a mistake when you staged your changes...

Reply:
done